### ISSUE TITLE
Unable to get data when the first column is empty

### ISSUE THREAD

**Issue by Paliking** [wwwiiilll/pyxlsb#3]
Hi,
thanks for the library.
It works well when the first column is not empty.

INPUT
```python
wb = pyxlsb.open_workbook(xlsb_file)
sheet = wb.get_sheet(sheet_name)
print(sheet.dimension)
for row in sheet.rows():
    do something
```

OUTPUT
```
dimension(r=0, c=1, h=3, w=2)
File "...packages\pyxlsb\worksheet.py", line 71, in rows
    row[item[1].c] = Cell._make([row_num, item[1].c, item[1].v])
IndexError: list assignment index out of range
```

I'm using 64-bit python 3 on Windows 10. 
pyxlsb v1.0.2
Do you have the same issue?
Thank you

**Comment by wwwiiilll** [comment]
Oh that's interesting, I never had a case with an empty first column.

With those dimension numbers I assume you have columns B and C filled, right?

**Comment by wwwiiilll** [comment]
Fixed in 1.0.3

**Comment by Paliking** [comment]
yes, exactly. 
thx for your fast fix.
